### ISSUE TITLE
Cross project +it:test seems to be broken with 1.3.5, works with 1.3.4

### ISSUE THREAD

**Issue by 35VLG84** [sbt/sbt#5336]
## steps

1. `git clone https://github.com/e257-fi/tackler.git`
2. `sbt "+it:test"` 

### SBT +it:test works with 1.3.4
```
tackler>  sbtVersion
[info] apiJS / sbtVersion
[info]  1.3.4
[info] apiJVM / sbtVersion
[info]  1.3.4
[info] core / sbtVersion
[info]  1.3.4
[info] cli / sbtVersion
[info]  1.3.4
[info] sbtVersion
[info]  1.3.4

tackler>  +it:test
[info] Setting Scala version to 2.12.10 on 4 projects.
[info] Excluded 1 projects, run ++ 2.12.10 -v for more details.
[info] Reapplying settings...
[info] Set current project to tackler (in build file:/home/35vlg84/devel/e257/accounting/tackler/)
[info] Compiling 1 Scala source to /home/35vlg84/devel/e257/accounting/tackler/cli/target/scala-2.12/classes ...
[info] Done compiling.
....
```
### SBT +it:test fails with 1.3.5

```
tackler>  sbtVersion
[info] apiJS / sbtVersion
[info]  1.3.5
[info] apiJVM / sbtVersion
[info]  1.3.5
[info] core / sbtVersion
[info]  1.3.5
[info] cli / sbtVersion
[info]  1.3.5
[info] sbtVersion
[info]  1.3.5

tackler>  +it:test
[info] Forcing Scala version to 2.12.10 on all projects.
[info] Reapplying settings...
[info] Set current project to tackler (in build file:/home/35vlg84/devel/e257/accounting/tackler/)
[error] Expected ';'
[error] No such setting/task
[error] apiJS / It / test
[error]                  ^
```

## problem

SBT command ` +it:test` fails with sbt version 1.3.5, it works with sbt 1.3.4

## expectation

SBT command ` +it:test` should work with sbt version 1.3.5

## notes

SBT build definition can  be found here: https://github.com/e257-fi/tackler

**Comment by eatkins** [comment]
This will be fixed in 1.3.6 by https://github.com/sbt/sbt/pull/5334.

**Comment by 35VLG84** [comment]
Hello, sbt 1.3.6 fixed this indeed, thank you!
